src/lm/kaldi-lmtable.cc

```diff
@@ -207,15 +208,24 @@ bool LmTable::ReadFstFromLmFile(std::istream &istrm,
     }
     // found valid line
     ngram_order = atoi(inpline.substr(pos1+5, pos2-(pos1+5)).c_str());
+    orders.push_back(ngram_order);
     if (ngram_order > max_ngram_order) {
       max_ngram_order = ngram_order;
     }
   }
+  std::sort(orders.begin(), orders.end());
```

Comment:
@jtrmal Currently I just checked if statistics of n-grams is provided. Is it necessary to check real count and the number from the header?